octobot_trading/exchanges/exchange_manager.py

```diff
@@ -350,7 +350,7 @@ def _load_config_symbols_and_time_frames(self):
         client = self.exchange.client
         if client:
             self.client_symbols = client.symbols
-            self.client_time_frames = list(client.timeframes.values()) if hasattr(client, "timeframes") else []
+            self.client_time_frames = list(client.timeframes) if hasattr(client, "timeframes") else []
```

Comment:
We keep only keys now right ?

Reply:
yes, list(dict) returns a list of its keys